assets/shared/smoothscroll.js

```
@@ -56,6 +56,11 @@
         } else {
           // Fix scrolling inaccuracy (always 1 or 2 pixels off without it)
           element.scrollTop += to.getBoundingClientRect().top - offset;
+          
+          // Invoke callback if any
+          if (cb) {
```

Hmm I see this as a developer error, since there's no other reason to pass a second argument to the function. If a developer were to pass something else than a function and `smoothScroll` failed silently, it'd be hard for them to debug the resulting issue.